### DISCUSSION TITLE
May I get a std::initializer_list from a nlohmann::json object?

### DISCUSSION THREAD

**Discussion by frokaikan** [nlohmann/json#3190] [Q&A] [answered]
Assuming I have a function 
`
f(std::initializer_list<int> list)
`
and an object
`
nlohmann::json j ({1, 2, 3});
`
Then how can I call function f with j WITHOUT converting j to std::vector (or other container except std::initializer_list)?
Which means, can I write code like
`
f(j);
`
Thanks.

**Comment by nlohmann** [comment, marked as answer]
This is not possible at the moment.

**Reply by frokaikan** [reply]
Ah, thanks.
